src/readii/data/label.py

```
@@ -0,0 +1,278 @@
+    # end string handling
+    else:
+        raise TypeError("Event column {outcome_column_label} is not a valid type. Must be a string, boolean, or numeric.")
+
+
+    return dataframe_with_standardized_outcome
+
+
+
+def addOutcomeLabels(feature_data_to_label:DataFrame,
+                     clinical_data:DataFrame,
+                     outcome_labels:list = ["survival_time_in_years", "survival_event_binary"]):
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)